src/applications/check-in/day-of/pages/TravelQuestion.jsx

```diff
@@ -16,20 +67,19 @@ const TravelQuestion = props => {
   );
 
   return (
-    <TravelPage
-      header={t('would-you-like-to-file-a-travel-reimbursement-claim-now')}
-      eyebrow={t('check-in')}
-      bodyText={
-        <p>
-          {t(
-            'answer-a-few-questions-to-find-out-if-you-can-file-your-claim-now',
-          )}
-        </p>
-      }
-      helpText={helpText}
-      pageType="travel-question"
-      router={router}
-    />
+    <>
```

Comment:
not a big deal, but why the addition of the empty tags?

Reply:
This is a mistake, I think initially I had the idea I was going to add more elements into this area and ended up not doing that